src/app/config/no-config/no-config.component.ts

```diff
@@ -166,12 +200,12 @@ export class NoConfigComponent implements OnInit {
     return true;
   }
 
-  public validateConfig(): void {
+  public async validateConfig(): Promise<any> {
```

Comment:
Should return `Promise<void>`
```suggestion
  public async validateConfig(): Promise<void> {
```

Reply:
Didn't know I could do this void there, I know there are other types but I didn't know void was one, this will be handy thanks.